### PR TITLE
feat: add @muxunorg/mxorg-client package

### PR DESCRIPTION
**Pull request by ao-x**
## Summary by CodeRabbit

- **Chores**
  - Added a new dependency `@muxunorg/mxorg-client` to the project. This prepares the app for future enhancements and integrations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17613,6 +17613,9 @@
     "el-table-fixed-header": {
       "version": "*"
     },
+    "@muxunorg/mxorg-client": {
```
Comment (fengmk2) [inline package.json:17616]: 请问这个包里面的内容是关于什么的呢？

**Comment by Zhengqbbb** [comment]
1. Is this a package? It looks like a resource.
2. Except for the version updated today, the previous version `v0.0.4` only contained image resources.
3. Can this PR description be supplemented ?

https://www.npmjs.com/package/@muxunorg/mxorg-client/v/0.0.4?activeTab=code

**Comment by Zhengqbbb** [comment]
🤔 Is there a blacklist?

image

File: package.json (hunk 1 above)
Comment (ao-x) [reply package.json:17616]: 是一个准备开源游戏加速器的前端部分的组件

**Review by fengmk2 — APPROVED** [review]

**Pull request merged**